e2e/support/helpers/e2e-relative-date-picker-helpers.js

```diff
@@ -47,8 +47,9 @@ function addStartingFrom({ value, unit }) {
 }
 
 function toggleCurrentInterval() {
-  popover().findByLabelText("Options").click();
-  popover().last().findByTestId("include-current-interval-option").click();
+  popover()
```

Comment:
If we replace `popover()` with `cy` (which we can, because we're using `data-testid`), then we'll also be able to use this util in `e2e/test/scenarios/filters/filter.cy.spec.js`

Reply:
If we decide to go with the testid, we have to force click.
Testid is tied to the input that is hidden.

I'll tidy up helpers by the end of the milestone 1.